README.rst

```diff
@@ -16,6 +16,16 @@ How to Use This Docker Image
 
 Spilo's setup assumes that you've correctly configured a load balancer (HAProxy, ELB, Google load balancer) that directs client connections to the master. There are two ways to achieve this: A) if the load balancer relies on the status code to distinguish between the healthy and failed nodes (like ELB), then one needs to configure it to poll the API URL; otherwise, B) you can use callback scripts to change the load balancer configuration dynamically.
 
+How to Build This Docker Image
+==============================
+
+    sudo pip3 install 'docker<3' docker-squash 
+    
+    postgres-appliance/build.sh --build-arg COMPRESS=true --tag $YOUR_TAG
```

Comment:
```bash
$ cd postgres-appliance
$ ./build.sh --build-arg COMPRESS=true --tag $YOUR_TAG .
```

Other build arguments and their default values:
* --build-arg WITH_PERL=false  # set to true if you want to install perl and plperl packages into image
* --build-arg PGVERSION="10"
* --build-arg PGOLDVERSIONS="9.3 9.4 9.5 9.6"
* --build-arg DEMO=false  # set to true if you want to build smallest possible image which will work only on kubernetes